medmodels/medrecord/querying.pyi

```diff
@@ -86,24 +86,23 @@ class EdgeOperand:
 class MultipleValuesOperand:
     def max(self) -> SingleValueOperand: ...
     def min(self) -> SingleValueOperand: ...
-    def first_where(
-        self, query: Callable[[SingleValueOperand], None]
-    ) -> SingleValueOperand: ...
-    def last_where(
-        self, query: Callable[[SingleValueOperand], None]
-    ) -> SingleValueOperand: ...
     def mean(self) -> SingleValueOperand: ...
     def median(self) -> SingleValueOperand: ...
     def mode(self) -> SingleValueOperand: ...
     def std(self) -> SingleValueOperand: ...
     def var(self) -> SingleValueOperand: ...
     def count(self) -> SingleValueOperand: ...
     def sum(self) -> SingleValueOperand: ...
+    def first(self) -> SingleValueOperand: ...
```

Comment:
Why did we need a query argument in the previous first method and not anymore?

Reply:
Because it was unnecessarily difficult to implement and this should have the same functionality, e.g.
```python
values = node.attribute("time")

value = values.first_where(lambda value: value.is_datetime())
```
should be the same thing as
```python
values = node.attribute("time")

values.is_datetime()

value = values.first()
```